vcr/persisters/filesystem.py

```diff
@@ -5,17 +5,25 @@
 from ..serialize import deserialize, serialize
 
 
+class CassetteNotFoundError(FileNotFoundError):
+    pass
+
+
+class CassetteDecodeError(ValueError):
+    pass
+
+
 class FilesystemPersister:
     @classmethod
     def load_cassette(cls, cassette_path, serializer):
         cassette_path = Path(cassette_path)  # if cassette path is already Path this is no operation
         if not cassette_path.is_file():
-            raise ValueError("Cassette not found.")
+            raise CassetteNotFoundError()
         try:
             with cassette_path.open() as f:
                 data = f.read()
         except UnicodeEncodeError as err:
-            raise ValueError("Can't read Cassette, Encoding is broken") from err
+            raise CassetteDecodeError("Can't read Cassette, Encoding is broken") from err
```

Comment:
I believe `except UnicodeEncodeError as err:` needs to be `except UnicodeDecodeError as err:`, instead.

(Even if loading were to decode and re-encoded certain bytes, I believe it's unlikely that that re-encoding would fail, if decoding worked the first time. So probably we don't need to catch both but only `UnicodeDecodeError`. What do you think?)

Reply:
Agreed, done!